src/Init/Init.c

```diff
@@ -249,3 +213,33 @@ Init_setup_sdram_config(Sdramc* const sdramc)
 
     Sdramc_setConfig(sdramc, &IS42S16100H_SDRAM_CONFIG);
 }
+
+inline static void
+Init_setup_sdram(void)
+{
+    Sdramc sdramc;
+    Init_setup_sdram_pio();
+    Init_setup_sdram_pmc();
+    Sdramc_init(&sdramc);
+    Sdramc_startup(&sdramc);
+    Init_setup_sdram_config(&sdramc);
+    Sdramc_performInitializationSequence(&sdramc, SystemConfig_DefaultCoreClock);
+}
+
+inline static void
+Init_setup_broker_lock()
+{
+    broker_lock = xSemaphoreCreateBinary();
```

Comment:
`xSemaphoreCreateBinaryStatic`?

Reply:
In the new commit I have provided the functionality of static mem. allocation and fixed this problem